### ISSUE TITLE
Add JSON support

### ISSUE THREAD

**Issue by dlibanori** [current/vuash#93]
Right now it is not so easy to build a command line application that interacts with vuash cause we have to parse HTML. I was thinking to add JSON support: vuash starts to accept and respond `application/json`.

**Comment by jonduarte** [comment]
Sounds great! That'll allow us (and others) to create other type of clients for the service.

**Comment by rmasoni** [comment]
Won't this affect security?

Right now, the encryption key is generated by the client and delivered through the URL fragment. Won't generating a JSON file end up delivering this data to the server?

**Comment by jonduarte** [comment]
No, because they'll send the encrypted data only. The secret key will stay on client side. Which means that the client will need to implement themselves the `encryption` part, and vuash will work solely as an storage for encrypted data.

**Comment by rmasoni** [comment]
Doesn't that defeat the purpose of the app? It would only serve as storage, leaving encryption and other stuff to the client, and we wouldn't be able to guarantee compliance with our standards.

**Comment by dlibanori** [comment]
But it doesn't. The back-end doesn't know if its content is encrypted or not. Its main purpose is **delete after read** and we can only confirm this. The client-side is totally changeable whether it be HTML or [CLI](https://en.wikipedia.org/wiki/Command-line_interface).
